examples/ab_test.py

```diff
@@ -93,6 +94,73 @@
     "--test_value", type=str, required=True, help="the feature value in test group."
 )
 
+parser.add_argument("--csv", type=str, help="csv output file")
+
+
+def get_percent_diff(test_val, control_val):
+    return ((test_val - control_val) / control_val) * 100.0
+
+
+def seconds_to_ms(seconds):
+    return seconds * 1000.0
+
+
+def print_metric(
+    performance_data, resolutions, title_list, metric="fps", metric_transformer=None
+):
+    for nproc, performance in performance_all.items():
+        header = f" Performance ({metric}) NPROC={nproc} "
+        print(f"{header:=^100}")
+        title = "Resolution".ljust(16)
+        title += "".join(t.ljust(24) for t in title_list)
+        print(title)
+        # break down by resolutions
+        for resolution, perf in zip(resolutions, performance):
+            row = f"{resolution} x {resolution}".ljust(16)
+            # break down by benchmark items
+            for t in title_list:
+                control_metric = perf[t][dr.ABTestGroup.CONTROL][metric]
+                test_metric = perf[t][dr.ABTestGroup.TEST][metric]
+                percent_diff = get_percent_diff(test_metric, control_metric)
+                if metric_transformer:
+                    control_metric = metric_transformer(control_metric)
+                    test_metric = metric_transformer(test_metric)
+
+                row += (
+                    f"{control_metric:6.1f}/{test_metric:6.1f} ({percent_diff:>6.1f}%)"
```

Comment:
Like I mentioned in the previous message. Do you think it is more reasonable (useful) to report speedup instead of percent_diff?

Reply:
Let us add an option to the program, "--speedup".
By default, we use percent_diff, otherwise display "speedup" (e.g., 3.6x) if "--speedup" is specified.